oreClient/src/main/assets/api.js

```diff
@@ -27,6 +28,11 @@ export class API {
                         withCredentials: true
                     },
                 }).done((data) => {
+                    //store.commit({
```

Comment:
Did you ever get this working?

Reply:
yeah that works, remove the comments and it should work instantly